src/main/java/gregtech/api/unification/material/properties/OreProperty.java

```diff
@@ -165,6 +167,15 @@ public List<Material> getOreByProducts() {
 
     @Override
     public void verifyProperty(MaterialProperties properties) {
+
+        for(Material material : oreByProducts) {
+            if (!material.hasProperty(PropertyKey.DUST) && !material.hasProperty(PropertyKey.FLUID)) {
```

Comment:
If the material has a fluid property but no dust property, this section is skipped, which can allow for some rather strange ore byproducts that aren't logged.

Reply:
That was not the original goal of this PR, but I can add a check for this